lib/tasks/assemble.js

```diff
@@ -1,7 +1,8 @@
 'use strict';
 
 const chalk = require('chalk');
-const install = require('rsvp').denodeify(require('npmi'));
+const pruner = require('rsvp').denodeify(require('electron-packager/prune').pruneModules);
+const { hasYarn } = require('electron-forge/dist/util/yarn-or-npm');
```

Comment:
I also think maybe here we should just use the `yarn-or-npm` package directly, rather than using a private API in `electron-forge`. AFAIKT `electron-forge` only wraps it so it can introduce an environment variable for its automated testing. We don't currently need that, and if we did, I'd rather have our own implementation so we don't break if `electron-forge` changes how they do it.